e2e/utils/zetacore.go

```diff
@@ -101,6 +101,10 @@ func WaitCctxsMinedByInboundHash(
 		// declare cctxs here so we can print the last fetched one if we reach timeout
 		var cctxs []*crosschaintypes.CrossChainTx
 
+		if i%10 == 0 {
```

Comment:
is this for debugging only ? 
if not it woudl be better to use the logger

Reply:
Let me restore these log prints for debugging a panic.